app/helpers/react_components/track/activity_ticker.rb

```diff
@@ -12,13 +12,19 @@ def to_s
 
       def initial_data
         last_metric = Metric.where(track_id: track.id, type: ALLOWED_METRIC_TYPES).last
```

Comment:
You could also do:

```suggestion
        Metric.where(track_id: track.id, type: ALLOWED_METRIC_TYPES).last&.to_broadcast_hash
```

Reply:
Yes. Do this. So the whole method can be replaced by what Erik put :)